frontend/talentsearch/.env.example

```diff
@@ -1,43 +0,0 @@
-APP_NAME="Talent Search"
-APP_ENV=local
-APP_KEY=
-APP_DEBUG=true
-TALENTSEARCH_APP_URL="http://localhost:8000/talent"
-APP_TIMEZONE=UTC
-
-TALENTSEARCH_APP_DIR="/talent"
-APPLICANTPROFILE_APP_DIR="/talent/profile"
-TALENTSEARCH_ASSET_URL="/talent"
-TALENTSEARCH_RECRUITMENT_EMAIL="recruitmentimit-recrutementgiti@tbs-sct.gc.ca"
-TALENTSEARCH_SUPPORT_EMAIL="gctalent-talentgc@support-soutien.gc.ca"
-
-NODE_ENV=development
-
-LOG_CHANNEL=stack
-LOG_SLACK_WEBHOOK_URL=
-
-DB_CONNECTION=pgsql
-DB_HOST=postgres
-DB_PORT=5432
-DB_DATABASE=gctalent
-DB_USERNAME=postgres
-DB_PASSWORD=password1234
-
-CACHE_DRIVER=file
-QUEUE_CONNECTION=sync
-
-API_URI="/graphql"
-
-OAUTH_POST_LOGOUT_REDIRECT="http://localhost:8000/logged-out"
-
-# Mock auth endpoint
-OAUTH_LOGOUT_URI="http://localhost:8000/oxauth/endsession"
```

Comment:
I took a look at the config file that was returned.  I think we've got failures to put in env variables on both build and post-deploy!
```
["OAUTH_POST_LOGOUT_REDIRECT", filterUnusable("$OAUTH_POST_LOGOUT_REDIRECT") ?? ""], 
["OAUTH_LOGOUT_URI", filterUnusable("$OAUTH_LOGOUT_URI") ?? ""], 
["FEATURE_ONGOING_RECRUITMENTS", filterUnusable("$FEATURE_ONGOING_RECRUITMENTS") ?? ""], 
["APPLICATIONINSIGHTS_CONNECTION_STRING", filterUnusable("$APPLICATIONINSIGHTS_CONNECTION_STRING")], 
["LOG_CONSOLE_LEVEL", filterUnusable("$LOG_CONSOLE_LEVEL") ?? ""], 
["LOG_APPLICATIONINSIGHTS_LEVEL", filterUnusable("$LOG_APPLICATIONINSIGHTS_LEVEL") ?? ""]]);
```
I bet the build problem is the one Tristan found.  That's not really a big deal for deployment though since the post-deploy script should be filling in variables from the app-svc.  That's bigger problem imo.

Reply:
Okay, I think everything is resolved now. 😅 

- [move .env to root](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5344/commits/016b06f63e96b3406bde22209817c9553e2ad253)
- [fix path to .env file](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5344/commits/6f07502cb4b93535f7e79513c42ad6f2b8e543ce)
- [mount env to maintenance container](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5344/commits/3a06f95471cc576c05390e00bc5e5dd9dbf85a94)
- [fix docker .env mount + add .env.example](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5344/commits/ff034f1cf8253a593da66041072b577f6183fc45)

Reply:
Looks good!  Then config.js file coming in properly for both talentsearch and iap now.